lib/index.js

```diff
@@ -15,6 +15,20 @@ function extend() {
     return result;
 }
 
+function debounce (func, wait, immediate) {
```

Comment:
Hey,

Yes, there's a 100ms delay. The reason I did this is because I need to
support IE9 and giving this delay helps achieve performance. We could add
another prop here to control this delay?

Regarding using lodash's module, if you want I can replace them and update
the PR.

On 22 May 2016 at 23:58, afram <notifications@github.com> wrote:

> In lib/index.js
> <https://github.com/eiriklv/react-masonry-component/pull/35#discussion_r64154153>
> :
>
> > @@ -15,6 +15,20 @@ function extend() {
> >      return result;
> >  }
> >
> > +function debounce (func, wait, immediate) {
>
> It might be better to use the lodash debounce module. Same goes for the
> extend function, but that's outside the scope of this PR
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/eiriklv/react-masonry-component/pull/35/files/a2dbbb9d490558ce3b50a0634347025554f9c78a#r64154153>
>



-- 
Best Regards
Sankalp Lakhina
Roposo.com <http://www.roposo.com/>

Relevant E Solutions Private Limited allows reasonable use of the e-mail
system. Views and opinions expressed in these communications do not
necessarily represent those of Relevant E Solutions Private Limited.  This
communication (including any attachments) is intended only for the person
or entity to which it is addressed and may contain confidential and/or
privileged material.  If you are not the intended recipient, any
disclosure, copying, distribution or use of the contents of this
communication is prohibited.  If you have received this in error, please
contact the sender immediately and delete this communication without
copying.